docs/validator/economics.md

```diff
@@ -89,45 +105,47 @@ NEAR validators are subject to market and game-theory dynamics, relating to:
 
 <blockquote class="info">
     <strong>did you know?</strong><br><br>
-    The protocol automatically re-stakes with the validator rewards at the end of each epoch, so you may expect up to 4.5% additional stake every year.
+    The protocol automatically re-stakes with the validator rewards at the end of each epoch. Meaning the amount staked will be constantly growing if not withdrawn by the validator / staking contract.
 </blockquote>
 
 **Distribution of the stake across validators**
-At the end of every `epoch`, NEAR Protocol assigns seats to validators: the calculation is based on the `proposal` in $NEAR tokens, and the total number of *seats* available - which is `100` per shard. 
+At the end of every `epoch`, NEAR Protocol assigns seats to validators for epoch after next: the calculation is based on the `proposal` in $NEAR, and the total number of *seats* available - which is `100` per shard. 
 
 By monitoring the `proposals`, validators can consistently estimate the stake in $NEAR per each *seat*.
-This number can be obtained by dividing the total at stake by the number of seats:
 
-| Total $NEAR at stake | Total seats | $NEAR per-seat |
-| --- | --- | --- |
-| 400 million | 100 | 4 million |
-| 400 million | 400 | 1 million |
-| 400 million | 800 | 0.25 million |
+Let's say you observe a set of proposals (and rollovers from previous epoch):
+
+| Validator | Stake | # Seats |
+| - | - | - |
+V1 |   1,000,000    | 48
+V2 |      500,000   | 24
+V3 |      300,000   | 14 
+V4 |      300,000 | 14
+--- | seat price is 20,500 |  --- |
+V5 |         20,000 | 0 |
+
+The *seat price* given this proposals is determined by finding such integer number that if you divide each validator's stake with rounding down (e.g. V5 20,000 / 20,500 with rounding down will be 0) - you will get required number of seats. This determines who will get their seat(s) and who's funds will be returned back to them.
+
+The *seat price* for current epoch can be obtained by dividing the total at stake by the number of seats, for example at 400 million $NEAR at stake and 100 total seats, the *seat price* will be 4 million.
 
 More details can be found in the validators section of the [Economics White Paper](https://nearprotocol.com/papers/economics-in-sharded-blockchain/#validators)
 
 **Transaction fees and inflation**
-NEAR Protocol's rewards follow the formula `total_supply * 0.05/730` of NEAR tokens at the end of every `epoch` (producing up to 5% of new tokens per year). However, the protocol *burns* `tx_fees` at the end of each block. Therefore, while rewards increase `total_supply` of $NEAR tokens, the fees *reduce* it. 
+NEAR Protocol's rewards follow the formula `total_supply * 0.045/730` of NEAR tokens at the end of every `epoch`. However, the protocol *burns* `tx_fees` at the end of each block. Therefore, while rewards increase `total_supply` of $NEAR tokens, the fees *reduce* it. 
```

Comment:
why 0.045?

Reply:
I didn't mention the treasury in my version, but this is more accurate. If we allocate 10% of new tokens to the treasury, which is 0.005 per year, the reward *for validators* is 0.045, not 0.05.
It's a convention: we may say that reward is 0.05, of which 0.005 is treasury and the rest validators; or say that 0.045 is the reward for validators omitting the rest.